xbmc/platform/linux/AppParamParserWebOS.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (C) 2005-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "AppParamParserWebOS.h"
+
+CAppParamParserWebOS::CAppParamParserWebOS() : CAppParamParserLinux()
+{
+}
+
+CAppParamParserWebOS::~CAppParamParserWebOS() = default;
+
+void CAppParamParserWebOS::Parse(char* argv[], int nArgs)
```

Comment:
If you cannot even specify args when launching the app what's the point in having a `Parse` override?

You can likely just override `ParseArg` and simply do nothing and not call back into `CAppParamParser::ParseArg`. That should be enough to ignore any passed args and not have to adjust the `Parse` method.

Reply:
You can provide args if you launch the app through the command line. This patch just prevents parsing the args that are added by the gui launcher by default.